test/core/test/jest-expect.test.ts

```diff
@@ -892,3 +893,87 @@ it('correctly prints diff with asymmetric matchers', () => {
 })
 
 it('timeout', () => new Promise(resolve => setTimeout(resolve, 500)))
+
+describe.only('custom equality testers', () => {
```

Comment:
you probably forgot to remove the `.only` on this one 😉

Reply:
Ohhh, Thanks~